SIS_import/.env_example

```diff
@@ -3,8 +3,10 @@ canvas_allowed_call_number=2900
 alert_email_address=EMAIL_ADDRESS
 
 # for sis_upload
+current_directory=PATH
```

Comment:
I don't see this value being used, is it needed?

Reply:
The value is used in sis_upload.rb:

`$currentDataDirectory = $currentDirectory + "data/"`

Reply:
Okay so just something that was missing. Thanks!